Makefile

```diff
@@ -78,10 +78,10 @@ destroy-box:
 remove-tmp:
 	rm -rf ./tmp
 	rm -rf ./.vagrant
+	rm -rf ./dev/tmp
 	rm -rf ./.minio.sys
-	rm -rf ./example/*/.terraform
-	rm -rf ./example/*/terraform.tfstate
-	rm -rf ./example/*/terraform.tfstate.backup
+	rm -rf ./example/**/.terraform
+	rm -rf ./example/**/.terraform*
```

Comment:
Do we need one like this as well:
`terraform*`
?

Reply:
Yeap. There are several files with different suffix:
```
terraform.tfstate
terraform.tfstate.backup

```

Reply:
Okey, then I think you should update with `rm -rf ./example/**/terraform*

Reply:
resolved https://github.com/fredrikhgrelland/terraform-nomad-presto/pull/87/commits/f98483005a53957c3b493f105ec92ef775b78c97